lib/puppet/type/jenkins_job.rb

```diff
@@ -64,6 +67,11 @@ def change_to_s(currentvalue, newvalue)
     defaultto true
   end
 
+  newproperty(:replace, :boolean => true, :parent => Puppet::Property::Boolean) do
```

Comment:
Puppet::Property::Boolean is totally broken. (Setting something that is true to false just doesn’t work) But... should this actually be a parameter instead of a property?